.github/workflows/nebula.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: docker test
         run: docker --help
       - name: Gradle build
```

Comment:
Can you rename this step to make it more obvious that it is exercising tests?

Reply:
I will change the gradle task to `build`